react/features/virtual-background/components/VideoBackgroundButton.js

```diff
@@ -72,7 +66,7 @@ class VideoBackgroundButton extends AbstractButton<Props, *> {
      * @returns {boolean}
      */
     _isDisabled() {
-        return this.props._videoMuted;
+        return false;
```

Comment:
IIRC you can drop the `_isDisabled` function altogether since the base implementation returns false.